src/lib/fetch.ts

```diff
@@ -138,7 +141,12 @@ async function _handleRequest(
   let result: any
 
   try {
-    result = await fetcher(url, requestParams)
+    result = await fetcher(url, {
+      ...requestParams,
+      // UNDER NO CIRCUMSTANCE SHOULD THIS OPTION BE REMOVED, YOU MAY BE OPENING UP A SECURITY HOLE IN NEXT.JS APPS
+      // https://nextjs.org/docs/app/building-your-application/caching#opting-out-1
```

Comment:
"YOU MAY BE OPENING UP A SECURITY HOLE IN NEXT.JS APPS"

there's nothing at all about that in the link provided. 🤔
maybe the comment can be expanded to actually explain what issue this would cause?

Reply:
It does in the context of NextJS, as the docs state:

 > The Data Cache is persistent across incoming requests and deployments unless you revalidate or opt-out.
 
 So we have to forcefully opt-out of the cache, as it appears that `GET /user` may be cached just because. There's no docs on whether it's cached by the JWT, or not. We reached out to Vercel -- 0 information on that.

Reply:
maybe add that to the comment? most users are likely going to do the same thing i did and look on that page you linked for something mentioning "security" directly.